sktime/utils/classifier.py

```diff
@@ -0,0 +1,59 @@
+import numpy as np
+from sklearn.base import BaseEstimator, ClassifierMixin
+
+from sktime.utils import utilities
+from sktime.utils.utilities import check_data
+
+
+class Classifier(BaseEstimator, ClassifierMixin):
```

Comment:
Please inherit from the already existing [`BaseClassifier`](https://github.com/alan-turing-institute/sktime/blob/master/sktime/classifiers/base.py), but you can of course refactor your class into a `BaseProximityClassifier` or something which inherits from `BaseClassifier`

Reply:
Can I add the predict method to `BaseClassifier`? The predict method is usually the same for every classifier

Reply:
Currently, all of the classifiers have their own predict methods as far as I know, so adding this without deleting the class specific ones won't do anything. Refactoring out the `predict` method seems sensible though, but not sure if we want to do this now, I guess we would have to do some testing again or write  more extensive unit tests first.

Reply:
Seems more hassle than it's worth, I've just moved the predict method to a static method and made classifiers call that, which still reduces ~15 lines to 1